bot_components/slack_comm.py

```diff
@@ -8,7 +8,7 @@
 """
 import json
 
-import bot_components
+from .. import bot_components
```

Comment:
I'm not up to speed on the docker side of things, though I would imagine if this project worked just like any other python package, then there should be no need to set up any directory other than `rtmbot.conf` specifying to load the botcommander plugin. Assuming botcommander is properly installed `pip install botcommand`, for example`, then rtmbot will be able to load it.

This, of course, would require a different means for loading in changes to the module as the source files for the package shouldn't be modified. I have some ideas to handle that too (i.e., loading optional modules out of the current working directory).